draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1293,8 +1293,16 @@ element of F.
 It is convenient for hash-to-curve and decompression to agree on a notion of
 sign, since this may permit simpler implementations.
 
-See {{bg-curves}} for a discussion of representing x as a vector; this
-representation is used in both of the variants immediately below.
+See {{bg-curves}} for a discussion of representing elements of field extensions
+as vectors; this representation is used in both of the sgn0 variants below.
+
+Note that any valid sgn0 function for field extensions must iterate over
+the entire vector representation of the input element.
+To see why, imagine a function sgn0\* that ignores the final entry in its
+input vector, and consider a field element x = (0, 0, 0, 0, x\_5).
```

Comment:
We probably could make `m=2` for this example -- there's three redundant zeros!

Reply:
Good point!